packages/cli/src/models/network/NetworkAppController.js

```diff
@@ -9,8 +9,14 @@ import { AppProjectDeployer, SimpleProjectDeployer } from './ProjectDeployer';
 const log = new Logger('NetworkAppController');
 
 export default class NetworkAppController extends NetworkBaseController {
+  get isLightweight() {
+    return this.packageFile.isLightweight && !this.appAddress;
```

Comment:
I think we can rely only on the app address

Reply:
isLightweight will return true by default soon!